### ISSUE TITLE
Actions broken in 1.13.4 when using an item controller

### ISSUE THREAD

**Issue by gordonkristan** [emberjs/ember.js#12340]
[A user on StackOverflow](http://stackoverflow.com/questions/32555752/ember-cli-nothing-handled-the-action) has discovered that Ember 1.13.4 introduced a bug with item controllers (as far as I can tell). Below are two JSBins with code that uses an item controller. In the first (1.13.3), the action is caught by the controller and everything works fine. In the second (1.13.4), the action goes un-handled causing an error to be thrown.

[1.13.3](http://emberjs.jsbin.com/wojiyakafe/1/edit?html,js,console,output)
[1.13.4](http://emberjs.jsbin.com/yoxejitole/1/edit?html,js,console,output)

The gist of the code:

```
{{#each item in list itemController='item'}}
    {{! This action will be caught by the controller in 1.13.3 but not in 1.13.4}}
    <a {{action 'something'}}>
        {{item.model}}
    </a>
{{/each}}
```

I tested in 1.13.10 and it seems to still be there.

**Comment by rwjblue** [comment]
Agreed, this is a regression. From a quick look at the changelog I would guess https://github.com/emberjs/ember.js/pull/11667 is related.

**Comment by gordonkristan** [comment]
`git bisect` tells me that bebbf26941ac1af316e316a86d2a43dd29e248fd is the offending commit. (I confirmed that the commit before it, f5573923662456f78385937a8d7faeb476cf0d30, works as expected.) Unfortunately I'm not familiar enough with that code to know how to fix it, but hopefully that's a good starting point for somebody that is.

**Comment by spinlock99** [comment]
I'm glad to see that this issue is getting some love. I think these bugs have the same root cause:

* https://github.com/emberjs/ember.js/issues/12327
* https://github.com/emberjs/ember.js/issues/12322

Please let me know if there's anything I can do to help. My team's looked into this pretty extensively and would love to help with testing or anything else that we can do.

**Comment by mmun** [comment]
This remains a bug in Ember 1.13.13: http://emberjs.jsbin.com/nuzowi/1/edit?html,js,console,output

**Comment by rwjblue** [comment]
At this point I have to declare failure here. I highly doubt that this issue will get fixed (if someone does figure out a fix and PR's against release-1-13 branch it will be considered). 2.6 and above will not support the legacy controller addon, and support for things like itemControllers are completely removed from the code base at this point. 

I'm sorry we couldn't get this regression fixed in 1.13.

**Comment by spinlock99** [comment]
I don't think it's terrible to just leave `itemControllers` out of 1.13. Breaks semver but I think most people following Ember's development know that there were significant changes after 1.10. 

My advice would be not to upgrade past 1.10 if you are using `itemControllers`. Refactor them out first and then continue upgrading.
